framework/include/cppmicroservices/Bundle.h

```diff
@@ -329,11 +329,11 @@ class US_Framework_EXPORT Bundle
    * <code>BundleContext</code> can be used by the caller to act on behalf
    * of this bundle.
    *
-   * If this bundle is not in the <code>STATE_STARTED</code> state, then this
-   * bundle has no valid <code>BundleContext</code> and this method will
-   * return a default constructed \c BundleContext object.
+   * If this bundle is not in the \c STATE_STARTING, \c STATE_ACTIVE, or
+   * \c STATE_STOPPING states, then this bundle has no valid \c BundleContext
+   * and this method will return an invalid \c BundleContext object.
    *
-   * @return A valid or invalid <code>BundleContext</code> for this bundle or.
+   * @return A valid or invalid <code>BundleContext</code> for this bundle.
```

Comment:
Yeah, I didn't do a thorough cleanup, just simplified what I came across and might have missed some. I suggest to do a doc cleanup separately.